src/test/scala/io/iohk/ethereum/mining/BlockGeneratorSpec.scala

```diff
@@ -52,6 +52,22 @@ class BlockGeneratorSpec extends FlatSpec with Matchers with PropertyChecks with
     fulBlock.right.foreach(b => ledger.executeBlock(b, blockchainStorages.storages, validators) shouldBe a[Right[_, Seq[Receipt]]])
   }
 
+  it should "filter out failing transactions" in new Envirnoment {
```

Comment:
Could you fix the type in Environment? Or better yet, for consistency, change it to `TestSetup`, as that's the name we use everywhere else.